webpack/makeconfig.js

```diff
@@ -69,7 +69,7 @@ module.exports = function(isDevelopment) {
         loaders: isDevelopment ? [
           'react-hot', 'babel-loader'
         ] : [
-          'babel-loader'
+          'babel-loader?optional[]=optimisation.react.inlineElements'
```

Comment:
Not sure whether is safe to have this optimisation enabled only for production. What if it's not stable? It could lead to hard to find bugs.